### PR TITLE
Remove UWP meta-package reference and replace with indirect dependencies

### PR DESCRIPTION
**Pull request by MattGal**
Updated to use the package's dependencies, as seen here:
 https://github.com/dotnet/core-setup/pull/244

@weshaggard @ericstj

### CODE REVIEW

File: src/Common/test-runtime/project.json [hunk 1]
```diff
@@ -22,7 +22,32 @@
     "netstandard1.3": {},
     "uap10.0": {
       "dependencies": {
-        "Microsoft.NETCore.UniversalWindowsPlatform": "5.2.1-beta-000547-00",
+        "Microsoft.NETCore.Portable.Compatibility": "1.0.2",
+        "Microsoft.NETCore.Runtime.CoreCLR": "1.0.3",
```
Comment (ericstj) [inline src/Common/test-runtime/project.json]: You're missing a reference to Microsoft.NETCore.

**Comment by ericstj** [comment]
CI is failing because microsoft.xunit.runner.uwp still brings in UWP meta-package 5.0.0 which included a reference to Microsoft.NETCore.Runtime which would pin all the contract package versions.  Add a direct reference to `"Microsoft.NETCore.Runtime": "1.0.1"` or update microsoft.xunit.runner.uwp to reduce its dependencies.

File: src/Common/test-runtime/project.json (hunk 1 above)
Comment (weshaggard) [inline src/Common/test-runtime/project.json]: That should probably be expanded as well.

Comment (MattGal) [inline src/Common/test-runtime/project.json]: Just did this.

**Comment by MattGal** [comment]
@dotnet-bot Test this please.

**Comment by weshaggard** [comment]
Do we know why microsoft.xunit.runner.uwp has a direct dependency to UWP 5.0 meta-package?

**Comment by MattGal** [comment]
@dotnet-bot Test this please

**Comment by MattGal** [comment]
@dotnet-bot  Test this please.

**Comment by MattGal** [comment]
@dotnet-bot Test this please

**Comment by ericstj** [comment]
I think your failures are happening because of overlapping dependencies.  NuGet layers the dependency sections in project.json so you have one section (the one that isn't framework specific) asking for a higher version than the other, but the validation task sees the lower section.

**Comment by MattGal** [comment]
@ericstj  Thanks for the help, I've rebased on latest auto-updated version commit and removed all the duplicate entries, here's hoping it will build.

**Comment by weshaggard** [comment]
LGTM

**Pull request merged**